WuV/WuV_slides.tex

```diff
@@ -6,6 +6,7 @@
 \usepackage{amsmath}
 \usepackage{amssymb}
 \usepackage{xkeyval}
+\usepackage{tikz-cd}

```

Comment:
For a commutative diagram I inserted on the slide describing querying and what translated syntactic functions (IsSummer) to semantic ones have to adhere to. The equation there contained a typo and also was hard to read; it's much nicer with a diagram now.